packages/opal-server/opal_server/data/data_update_publisher.py

```diff
@@ -87,3 +90,16 @@ def publish_data_updates(self, update: DataUpdate):
             entries=logged_entries,
         )
         self._publisher.publish(all_topic_combos, update)
+
+    def create_polling_updates(self, sources: ServerDataSourceConfig):
+        #For every entry with a non zero period update interval, bind an inverval to it
+        updaters = []
+        if hasattr(sources, "entries"):
+            for source in filter(lambda s: s.periodic_update_interval > 0, sources.entries):
```

Comment:
This does show your age with Python 😅.
It would be better (more Pythonic, performant and readable to use list comprehension here).

https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions

Reply:
Better to check it's not None instead of bigger than zero; per than Zen of Python ("explicit it better than implicit")